scripts/dev/generate_epJSON_schema/idd_parser.py

```diff
@@ -110,15 +111,15 @@ def parse_idd(data):
         elif token == TOKEN_EXCLAMATION:
             eat_comment(data)
         elif token == TOKEN_GROUP:
-            next_token(data)
-            eat_comment(data)
+            current_group_name = parse_group(data).replace('\\group ', '')
```

Comment:
@mbadams5 I know this isn't perfect, and that I should just leverage one of your functions to "eat" the `\group ` part, but this does work properly, so should just take a small tweak to refine it.